cschwitalla/Ligandomics_Analysis/Ligandomics_Analysis.R

```diff
@@ -1,20 +1,28 @@
+################################################################################
+###                                Purpose                                   ###
+################################################################################
 
-
+################################################################################
+###                               Manual steps                               ###
+################################################################################
 setwd("/Users/cschwitalla/Documents/Immunopeptidomics/")
 
-library("stringr")
-library("RColorBrewer")
-library("tidyr")
-library("dplyr")
-library("tibble")
-library("ggplot2")
-library("viridis")
-library("VennDiagram")
-library("tuple")
-library("org.Hs.eg.db") 
-library("ggVennDiagram")
-library("readxl")
+# inputdir <-
+# outputdir <-
+source("functions_ligandomics.R")
+################################################################################
+###                             Load libraries                               ###
+################################################################################
+required_Libs <- c("tidyr","readxl", "ggVennDiagram", "dplyr", "stringr", "tibble", 
```

Comment:
Look at the comments of GitHub actions:
Line 16 contains multiple warnings:
1. The text is exceeding the 80 character limit
2. You write a variable `required_Libs`, I would change this to `required_libs`
3. `"tidyr","readxl"` --> `"tidyr", "readxl"`
4. you end the sentence with a space, that needs to be removed 😉